pymc3/model.py

```diff
@@ -994,6 +994,14 @@ def __init__(self, type=None, owner=None, index=None, name=None,
             self.model = model
 
             transformed_name = "{}_{}_".format(name, transform.name)
+            bound_dict = {'a':None, 'b':None}
+            for key in bound_dict:
+                try:
+                    bound_dict[key] = transform.__dict__[key].eval()
+                except KeyError:
+                    pass
+            if bound_dict['a'] is not None or bound_dict['b'] is not None:
+                transformed_name += '({},{})_'.format(bound_dict['a'], bound_dict['b'])                  
             self.transformed = model.Var(
```

Comment:
Can't we monkey-patch `lower` and `upper` onto `self.transformed`? While not clean, it would be cleaner than this.